.github/workflows/transports-release.yml

```
@@ -30,6 +37,17 @@ jobs:
         env:
           COMMIT_MSG: ${{ github.event.head_commit.message }}
         run: |
+          echo "📝 Checking release trigger..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)